cmd/motor-lib/wasm.go

```diff
@@ -0,0 +1,24 @@
+//go:build wasm
+// +build wasm
+
+package motor
+
+import (
+	"context"
+
+	"github.com/sonr-io/sonr/pkg/config"
+	"github.com/sonr-io/sonr/pkg/host"
+)
+
+func main() {
+	config := config.DefaultConfig(config.Role_MOTOR)
+	_, err := host.NewWasmHost(context.Background(), config)
```

Comment:
does not appear we do anything with the returned "host" for the wasm target

Reply:
We dont have p2p interaction methods or highway interaction methods yet